lib/jobs/ansible-job.js

```diff
@@ -65,12 +86,29 @@ function ansibleJobFactory(
             return self._publishAnsibleResult(self.routingKey, data);
         })
         .then(function() {
+            return fs.readFileAsync(self.outFileName, 'utf8');
+        })
+        .then(function(contents) {
+            assert.arrayOfString(self.context.ansibleResultFile);
+            self.context.ansibleResultFile.push(contents);
+            fs.unlink(self.outFileName, function(err) {
```

Comment:
Yeah I also read the code as not concerning the job success or execution with the results of unlink, which in this case I think is fine, especially since you log failures.